src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const oneDayPay = 40;
+  let prise = 0;
+
+  if (days < 3) {
+    prise = days * oneDayPay;
+  }
+
+  if (days >= 3 && days < 7) {
+    prise = days * oneDayPay - 20;
+  }
+
+  if (days >= 7) {
+    prise = days * oneDayPay - 50;
+  }
+
```

Comment:
move magic numbers in the variables, also if you will return value inside your if block, you can avoid one of them